cf/api/services.go

```diff
@@ -255,6 +256,22 @@ func (repo CloudControllerServiceRepository) FindServicePlanByDescription(planDe
 	return planGuid, apiErr
 }
 
+func (repo CloudControllerServiceRepository) ListServicesFromManyBrokers(brokerGuids []string) (services []models.ServiceOffering, err error) {
```

Comment:
This is how most functions in the file (including the one we were modeling ours off of) were returning, so we thought that was desirable. We can go ahead and change it if that's the convention you're moving toward.